examples/oktapam_gw_setup/gw_setup_example.tf

```diff
@@ -0,0 +1,69 @@
+terraform {
```

Comment:
nit You can separate the provider config part in a file main.tf. For oktapam provider version, we can just put >= 0.1.4.

Reply:
Done